_automating-configurations/workflow-builder.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: default
+title: Build workflows
+nav_order: 15
+---
+
+# Build workflows with OpenSearch Flow
+
+The OpenSearch Flow plugin in OpenSearch Dashboards lets you iteratively build and test use cases leveraging ingest and search pipelines. It aims to simplify the complexity around building different AI/ML use cases using ML inference processors, such as vector search and retrieval-augmented generation (RAG). Behind the scenes, the plugin uses the [Flow Framework OpenSearch Plugin]({{site.url}}{{site.baseurl}}/automating-configurations/index/) for resource management. Once you have built out a use case to your satisfaction, you can export the [Workflow Template]({{site.url}}{{site.baseurl}}/automating-configurations/workflow-templates/) to re-create identical resources across multiple clusters.
+
+## Background: Ingest pipelines, search pipelines, and ML models
+
+[Ingest pipelines]({{site.url}}{{site.baseurl}}/ingest-pipelines/) and [search pipelines]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/index/) are used for configuring data transforms at different stages of ingest and search operations using individual processors, all run within OpenSearch. An _ingest pipeline_ is composed of a sequence of _ingest processors_, while a _search pipeline_ is composed of a sequence of _search request processors_ and/or _search response processors_. You can stitch together processors and build custom pipelines to fit your data processing needs.
+
+In general, these pipelines allow for modifying data at 3 different stages.
+
+1. **Ingest**: transform your documents before they are ingested in an index.
+2. **Search request**: transform the search request before executing search against an index.
+3. **Search response**: transform the search response (and documents) after search is executed, but before the response is returned to the user.
+
+The [ML Commons OpenSearch Plugin]({{site.url}}{{site.baseurl}}/ml-commons-plugin/) lets you [integrate models hosted on third-party platforms]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/index/), and use their inference capabilities directly within OpenSearch. Both ingest and search pipelines offer [ML inference processors]({{site.url}}{{site.baseurl}}/ingest-pipelines/processors/ml-inference/), allowing you to leverage remote model inference within your pipelines at both ingest and search time. This plugin helps in building, testing, and visualizing ingest and search pipelines for your particular use case, including the use of ML inference processors.
+
```

Comment:
instead of "This plugin helps in building, testing, and visualizing ingest and search pipelines for your particular use case, including the use of ML inference processors.", can we say "OpenSearch Flow plugin helps in building, testing, and visualizing ingest and search pipelines for your particular use case, including the use of ML inference processors."

Reply:
updated